hawkbit-rest/hawkbit-ddi-resource/src/main/java/org/eclipse/hawkbit/ddi/rest/resource/DdiRootController.java

```diff
@@ -621,4 +625,105 @@ private static UpdateMode getUpdateMode(final DdiConfigData configData) {
         return null;
     }
 
+
```

Comment:
Please modify the "getControllerBasedeploymentAction" and "postBasedeploymentActionFeedback" method to deny the request in case the provided actionId is in the WFC state and not RUNNING.

Reply:
Done. HttpStatus not found is applied in that case.